### ISSUE TITLE
Colour Picker needs extra French text

### ISSUE THREAD

**Issue by james-rae** [ramp4-pcar4/highcharts-editor#163]
## Description

Looks like the colour picker is not being fed the custom french stuff

## Steps to reproduce

1. Start making a chart in French mode
2. Get to the Customize Chart - Data Series part of the builder
3. Open up the colour picker
4. Right click and inspect the knob on the hue slider.
5. See the descriptive text is `Hue` and not `Teinte`

## Secret Knowledge

The implementation in RAMP has the extra sauce provided. Should be able to copy the same technique. 
If things explode, might be related to https://github.com/ramp4-pcar4/ramp4-pcar4/issues/2400

```tsx
<ColorPicker
    v-if="layerInfo?.configOptions.includes('colour')"
    alpha-channel="hide"
    :visible-formats="['hex']"
    default-format="hex"
    :id="randomId + '-hue-slider'"
    :color="colour"
    @color-change="updateColour"
>
    <template #hue-range-input-label>
        <span class="sr-only">{{ t('wizard.configure.colour.hue') }}</span>
    </template>

    <template #copy-button>
        <span class="sr-only">{{ t('wizard.configure.colour.copy') }}</span>

        <svg
            aria-hidden="true"
            xmlns="http://www.w3.org/2000/svg"
            width="15"
            height="15"
            viewBox="0 0 15 15"
        >
            <path
                d="M5 0v2H1v13h12v-3h-1v2H2V5h10v3h1V2H9V0zm1 1h2v2h3v1H3V3h3z"
                fill="currentColor"
            />

            <path
                d="M10 7v2h5v2h-5v2l-3-3zM3 6h5v1H3zm0 2h3v1H3zm0 2h3v1H3zm0 2h5v1H3z"
                fill="currentColor"
            />
        </svg>
    </template>
</ColorPicker>
```
